insitu/templates/product_detail.html

```diff
@@ -0,0 +1,129 @@
+{% extends '_layout.html' %}
+{% load static %}
+
+{% block style %}
+  <style>
+    .popover {
+      min-width: 40%;
+      max-width: 100%;
+    }
+  </style>
+{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Product details</h1>
+    </div>
+  </div>
+
+  <div class="row">
+    <table id="detail-table" class="table borderless col-sm-12">
+      <tbody>
+      <tr class="row">
+        <th class="col-md-4 col-sm-6">
+          Name
+        </th>
+        <td class="col-md-8 col-sm-6">
+          {{ product.name }}
+        </td>
```

Comment:
This might look better if you use a single line for the `th` and a single line for the `td`.